istqb-ctel-itp-atp-complete-guide.md

```
@@ -0,0 +1,1874 @@
+  ① 全般的プロセス範囲（General Process Scope）：
+     - テストプロセス以外のプロセス（プロジェクト管理等）の範囲
+     - テスト以外でも改善が必要な場合を特定
+
+  ② テストプロセス範囲（Test Process Scope）：
+     - テストプロセスのどの部分を対象とするか
+     - 全体を広く改善 or 特定の側面（例：テスト計画）のみ
+
+  ③ プロジェクト範囲（Project Scope）：
+     プロジェクト/プログラム中心 vs 組織中心：
+
+     プロジェクト中心（Program/Project-centric）：
```

markdownlint-cli2 (0.22.1)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)